src/sc2api/CMakeLists.txt

```diff
@@ -26,7 +26,35 @@ set(sc2api_sources
     sc2_unit.cc
     "typeids/sc2_${SC2_VERSION}_typeenums.cpp")
 
-add_library(sc2api STATIC ${sc2api_sources})
+set(header_path "${PROJECT_SOURCE_DIR}/include/sc2api/")
+set(sc2api_headers
+    ${header_path}/sc2_action.h
```

Comment:
I think you should apply list transform to properly modify all paths and reduce repetition.
See https://cmake.org/cmake/help/latest/command/list.html#transform

Reply:
Not needed after the recent changes.